pylint/checkers/refactoring/implicit_booleaness_checker.py

```diff
@@ -139,6 +152,80 @@ def visit_unaryop(self, node: nodes.UnaryOp) -> None:
     @utils.only_required_for_messages("use-implicit-booleaness-not-comparison")
     def visit_compare(self, node: nodes.Compare) -> None:
         self._check_use_implicit_booleaness_not_comparison(node)
+        self._check_compare_to_empty_string(node)
+        self._check_compare_to_zero(node)
+
+    def _check_compare_to_zero(self, node: nodes.Compare) -> None:
```

Comment:
Those two functions are highly duplicated, they could be merged if we decide that everything can be merged in this checker.

Reply:
I did that.